features/shopify-markets-js-api.md

```diff
@@ -0,0 +1,340 @@
+# Shopify Markets Workaround
+
+{% hint style="info" %} Please note that this is a temporary workaround for loading recommendations based on Shopify Market until we release a complete end to end support. This workaround avoids any manual setup to be done on merchant's store. Instead, it involves minor changes to the existing recommendation templates.
+{% endhint %}
+
+## JS API to the rescue
+
+### Drawbacks of current approach
+In order to workaround product pricing for Shopify Markets, the existing approach follows the changes outlined [here](/features/multi-currency-support.md#fetch-product-prices-in-presentment-currency-legacy). This involves updating Shopify theme liquid template manually in order to enable Shopify Markets. The approach is also limited to loading product prices for the market. 
```

Comment:
Not included those snippet inline here for the following reasons
1. It will duplicate the snippet
2. Merchants are already well aware of the page. So, though it shouldn't be much of hassle. Moreover, the actual snippet is not about SM workaround and need not be included inline here
Added the link  here so that the merchant will know the JS API is actually an alternative approach for the legacy pricing logic.

Reply:
I think this page still needs to be clearer about the required theme changes.